src/olympia/devhub/views.py

```diff
@@ -1385,7 +1383,7 @@ def decorator(f):
         @functools.wraps(f)
         def wrapper(request, *args, **kw):
             step = outer_step
-            max_step = 6
+            max_step = 5
```

Comment:
`Addon` already has an `is_incomplete()` method... that uses the `devhub.models.SubmitStep` method... which reminds me, can we get rid of that model as well ? :)